index.html

```diff
@@ -144,65 +144,101 @@ <h2 class="heading2">Project name goes here</h2>
-            <li>Capibara</li>
-            <li>Selenium</li>
-          </ul>
-          <img class="icon-bg" src="images/ic_framework.svg" alt="#">
-        </li>
-        <li class="yellow about heading3"> Skills
-          <ul class="para2 side-li">
-            <li>Codekit</li>
-            <li>Codepen</li>
-            <li>Github</li>
-            <li>Gitlab</li>
-            <li>TErminal</li>
-          </ul>
-          <img class="icon-bg" src="images/ic_skills.svg" alt="#">
-        </li>
-      </ul>
-      </section>
-      <form action="https://formspree.io/f/xnqwnerg" method="post">
-        <ul>
-          <li>
-            <label for="name">Your name
-              <input type="text" id="name" name="user-name" placeholder="Text input" maxlength="30" required>
-            </label>
-          </li>
-          <li>
-            <label for="mail">Your email
-              <input type="email" id="mail" name="user-email" required>
-            </label>
+          <li class="blue about heading3">
+            Languages
+            <ul class="para2 side-li">
+              <li>JavaScript (ES5/ES6)</li>
+              <li>Ruby</li>
+              <li>HTML</li>
+              <li>CSS</li>
+            </ul>
+            <img class="icon-bg" src="images/ic_languages_mob.svg" alt="#" />
           </li>
-          <li>
-            <textarea id="msg" name="user-message" placeholder="Enter text here" maxlength="500" required></textarea>
+          <li class="red about heading3">
+            Frameworks
+            <ul class="para2 side-li">
+              <li>Bootstrap</li>
+              <li>Ruby on rails</li>
+              <li>RSpec</li>
+              <li>Capibara</li>
+              <li>Selenium</li>
+            </ul>
+            <img class="icon-bg" src="images/ic_framework.svg" alt="#" />
           </li>
-          <li>
-            <button type="submit" class="buttons-primary-dark">Start collaboration</button>
+          <li class="yellow about heading3">
+            Skills
+            <ul class="para2 side-li">
+              <li>Codekit</li>
+              <li>Codepen</li>
+              <li>Github</li>
+              <li>Gitlab</li>
+              <li>TErminal</li>
+            </ul>
+            <img class="icon-bg" src="images/ic_skills.svg" alt="#" />
           </li>
         </ul>
-      </form>
+      </section>
+      <section class="form-container">
+        <img
+          class="nav logo icon"
+          src="images/image_geometry_1.svg"
+          alt="logo"
+        />
+        <form
+          action="https://formspree.io/f/xnqwnerg"
+          method="post"
+          class="white-bg"
+        >
+          <h2 class="heading">Get started</h2>
+          <p class="para">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+            eiusmod tempor incididunt ut labore et dolore magna.
+          </p>
+          <ul>
+            <li class="form-li" tabindex="0">
```

Comment:
as I contained both the label and input in an li and reset all their properties, I wanted to give it (: focus) to li, but it only activates when I click on the frame or the padding of the li, and it doesn't activate f I click on the input or label. Are there any workarounds other than using JS?

Thank you